File: app/src/main/java/com/rong/drop/framework/database/RealmHelper.java
package com.rong.drop.framework.database;

import javax.inject.Inject;
import javax.inject.Singleton;
import io.realm.Realm;

/**
 * Created by chen.huarong on 2017/12/11.
 */

public class RealmHelper {

    private static RealmHelper mInstance;
    private Realm mRealm;

    @Singleton
    public RealmHelper() {

    }
}

File: app/src/main/java/com/rong/drop/framework/database/RealmHelper.kt
package com.rong.drop.framework.database

import javax.inject.Singleton
import io.realm.Realm

/**
 * Created by chen.huarong on 2017/12/11.
 */

@Singleton
class RealmHelper private constructor() {

    var realm = Realm.getDefaultInstance()


}
